### PR TITLE
[vulkan-headers] Update port to latest 1.3.234 version

### PR DESCRIPTION
**Pull request by Honeybunch**
**Describe the pull request**

- #### What does your PR fix?
  Updates vulkan-headers

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  All

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
 Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

File: ports/vulkan-headers/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vulkan-headers",
-  "version": "1.3.224",
+  "version": "1.3.233",
   "description": "Vulkan header files and API registry",
   "homepage": "https://github.com/KhronosGroup/Vulkan-Headers"
 }
```
[the remark this answers is not in the document]
Comment (MonicaLiu0311) [reply ports/vulkan-headers/vcpkg.json:6]: A `license` is required.

[the remark this answers is not in the document]
Comment (Honeybunch) [reply ports/vulkan-headers/vcpkg.json]: Oops don't know how that happened. Fixed

**Comment by juan-lunarg** [comment]
@Honeybunch as of [v1.3.234](https://github.com/KhronosGroup/Vulkan-Headers/tree/v1.3.234) there is now proper find_package support for Vulkan-Headers.

```
# https://github.com/KhronosGroup/Vulkan-Headers/blob/v1.3.234/BUILD.md#usage-in-cmake
find_package(VulkanHeaders REQUIRED CONFIG)

target_link_libraries(foobar PRIVATE Vulkan::Headers)
```

**Comment by juan-lunarg** [comment]
This `find_package` support is actively tested via the validation layers and the loader.

See https://github.com/KhronosGroup/Vulkan-ValidationLayers/commit/e09172e79c0b842a1ea2f5357e9f9d35d2be86fd as an example.

**Comment by Honeybunch** [comment]
Bumped to latest version

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**